packages/treasury/src/vaultManager.js

```diff
@@ -231,17 +233,17 @@ export function makeVaultManager(
     // the payout object
     return harden({
       uiNotifier: notifier,
-      invitationMakers: {
+      invitationMakers: Far('why are these methods upper case', {
```

Comment:
@katelynsills   are invitationMakers used with the wallet?  Does it need their keys to be Keyword-style?

Reply:
Given https://github.com/Agoric/agoric-sdk/issues/3503 and a plausible name, I'm resolving this.

Reply:
Yup, invitationMakers are used within dapp-svelte wallet and they need to be Keyword-style so that the wallet is protected against a contract developer.